src/azure-cli-core/setup.py

```diff
@@ -53,6 +53,8 @@
     'knack~=0.9.0',
     'msal-extensions>=0.3.1,<0.4',
     'msal>=1.16.0,<2.0.0',
+    'msrestazure~=0.6.4',
+    'packaging>=20.9,<22.0',
```

Comment:
The version bound is copied from #20661. We set the upper bound simply because we don't want unexpected breaking change, which is a value point today.